### PR TITLE
feature base activity and map fragment

### PR DESCRIPTION
**Pull request by ManekenT**
CLOSES https://github.com/monsieur-x/android-app/issues/15

### CODE REVIEW

File: app/src/main/java/io/monsieurx/app/android/view/MainActivity.java [hunk 1]
```diff
@@ -1,15 +1,168 @@
...
+
+        Toolbar myToolbar = (Toolbar) findViewById(R.id.actionBar);
+        setSupportActionBar(myToolbar);
+
+        _navListTitles = getResources().getStringArray(R.array.nav_bar_titles);
+        _drawerLayout = (DrawerLayout) findViewById(R.id.drawerLayout);
+        _drawerPane = (RelativeLayout) findViewById(R.id.drawerPane);
+        _navList = (ListView) findViewById(R.id.navList);
+
+        // Set the adapter for the list view
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/MainActivity.java]: remove

File: app/src/main/java/io/monsieurx/app/android/view/MainActivity.java [hunk 2]
```diff
@@ -1,15 +1,168 @@
...
+        _navList.setAdapter(new ArrayAdapter<String>(this,
+                R.layout.drawer_list_item, _navListTitles));
+        // Set the list's click listener
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/MainActivity.java]: remove

File: app/src/main/java/io/monsieurx/app/android/view/MainActivity.java [hunk 3]
```diff
@@ -1,15 +1,168 @@
...
+        _navList.setOnItemClickListener(new DrawerItemClickListener() {
+
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/MainActivity.java]: remove

File: app/src/main/java/io/monsieurx/app/android/view/MainActivity.java [hunk 4]
```diff
@@ -1,15 +1,168 @@
...
+        });
+
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/MainActivity.java]: remove

File: app/src/main/java/io/monsieurx/app/android/view/MainActivity.java [hunk 5]
```diff
@@ -1,15 +1,168 @@
...
+                super.onDrawerOpened(drawerView);
+                getSupportActionBar().setTitle(R.string.app_name);
+            }
+        };
+
+        // Set the drawer toggle as the DrawerListener
+        _drawerLayout.setDrawerListener(_drawerToggle);
+
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        //getSupportActionBar().setIcon(R.mipmap.ic_launcher);
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/MainActivity.java]: remove

File: app/src/main/java/io/monsieurx/app/android/view/MainActivity.java [hunk 6]
```diff
@@ -1,15 +1,168 @@
...
+                return true;
+            case R.id.action_item_2:
+                Toast.makeText(this, "Item 2", Toast.LENGTH_SHORT).show();
+                return true;
+
+            default:
+                // If we got here, the user's action was not recognized.
+                // Invoke the superclass to handle it.
+                return super.onOptionsItemSelected(item);
+
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/MainActivity.java]: remove

File: app/src/main/java/io/monsieurx/app/android/view/MapFragment.java [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+import android.view.ViewGroup;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.OnMapReadyCallback;
+
+import io.monsieurx.app.android.R;
+
+public class MapFragment extends Fragment {
+
+
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/MapFragment.java]: remove

File: app/src/main/java/io/monsieurx/app/android/view/MapFragment.java [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    public MapFragment() {
+        // Required empty public constructor
+    }
+
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/MapFragment.java]: remove

File: app/src/main/java/io/monsieurx/app/android/view/MapFragment.java [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+        final FragmentManager fragmentManager = getFragmentManager();
+        final FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
+        fragmentTransaction.replace(R.id.mapPlaceholder, com.google.android.gms.maps.MapFragment.newInstance()).commit();
+
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_map, container, false);
+    }
+
+
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/MapFragment.java]: 2x remove

File: app/src/main/java/io/monsieurx/app/android/view/PreferencesFragment.java [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+import android.view.ViewGroup;
+
+import io.monsieurx.app.android.R;
+
+public class PreferencesFragment extends Fragment {
+
+    public PreferencesFragment() {
+        // Required empty public constructor
+    }
+
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/PreferencesFragment.java]: remove

File: app/src/main/java/io/monsieurx/app/android/view/PreferencesFragment.java [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
```
Comment (nstaeger) [inline app/src/main/java/io/monsieurx/app/android/view/PreferencesFragment.java]: Besserer Kommentar?

File: app/src/main/res/layout/activity_main.xml [hunk 12]
```diff
@@ -3,15 +3,48 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+
+        <!-- The navigation drawer -->
+        <RelativeLayout
+            android:layout_width="280dp"
+            android:layout_height="match_parent"
+            android:id="@+id/drawerPane"
+            android:layout_gravity="start">
+
```
Comment (nstaeger) [inline app/src/main/res/layout/activity_main.xml]: remove

File: app/src/main/res/layout/fragment_map.xml [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+            android:textSize="30sp"
+            android:textColor="#000000" />
+    </LinearLayout>
+
+    <Button
+        android:layout_width="100sp"
+        android:layout_height="100sp"
+        android:text="@string/catch_button_label"
+        android:id="@+id/catchButton"
+        android:background = "@drawable/roundedbutton"
```
Comment (likesmusik) [inline app/src/main/res/layout/fragment_map.xml]: no spaces

File: app/src/main/res/drawable/roundedbutton.xml [hunk 14]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/colorAccent" />
+    <corners android:bottomRightRadius="50dp"
+        android:bottomLeftRadius="50dp"
+        android:topRightRadius="50dp"
+        android:topLeftRadius="50dp"/>
+</shape>
```
Comment (jjarzbach) [inline app/src/main/res/drawable/roundedbutton.xml]: new line

File: app/src/main/res/layout/drawer_list_item.xml [hunk 15]
```diff
@@ -0,0 +1,11 @@
...
+    android:id="@android:id/text1"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:textAppearance="?android:attr/textAppearanceListItemSmall"
+    android:gravity="center_vertical"
+    android:paddingLeft="16dp"
+    android:paddingRight="16dp"
+    android:textColor="@color/colorText"
+    android:background="?android:attr/activatedBackgroundIndicator"
+    android:minHeight="?android:attr/listPreferredItemHeightSmall"/>
```
Comment (jjarzbach) [inline app/src/main/res/layout/drawer_list_item.xml]: new line

File: app/src/main/res/values/strings.xml [hunk 16]
```diff
@@ -1,3 +1,10 @@
 <resources>
     <string name="app_name">Monsieur X</string>
+    <string name="drawer_open">Open Drawer</string>
```
Comment (likesmusik) [inline app/src/main/res/values/strings.xml]: Seperate in blocks

**Pull request merged**